core/core-configuration-layer.el

```diff
@@ -155,60 +143,55 @@ NAME."
         (replace-match name t)))
     (save-buffer)))
 
-(defun configuration-layer//get-contrib-category-dirs ()
-  "Return a list of all absolute paths to the contribution categories stored
-in `configuration-layer-contrib-categories'"
-  (mapcar
-   (lambda (d) (expand-file-name
-                (concat configuration-layer-contrib-directory (format "%s/" d))))
-   configuration-layer-contrib-categories))
+(defun configuration-layer//is-layer-p (path)
+  "Return non-nil if the path given is a valid configuration layer."
+  (when (file-directory-p path)
+    (let ((files (directory-files path)))
+      (or (member "packages.el" files)
```

Comment:
Not correct, a layer can have just a `config.el` file or just a `keybindings.el` file.